src/drawers/lineDrawer.ts

```diff
@@ -26,4 +34,45 @@ export class Line extends Drawer {
   public selectionForIndex(index: number): SimpleSelection<any> {
     return d3.select(this.selection().node());
   }
+
+  /**
+   *
+   * @param data Data to draw. The data will be passed through the line factory in order to get applied
+   * onto the canvas.
+   * @param step
+   * @private
+   */
+  protected _drawStepCanvas(data: any[][], step: AppliedDrawStep) {
+    const context = this.canvas().node().getContext("2d");
+
+    const d3Line = this._d3LineFactory();
+
+    const attrToAppliedProjector = step.attrToAppliedProjector;
+    const resolvedAttrs = Object.keys(attrToAppliedProjector).reduce((obj, attrName) => {
+      obj[attrName] = attrToAppliedProjector[attrName](data, 0);
+      return obj;
+    }, {} as { [key: string]: any | number | string });
+
+    context.save();
```

Comment:
What would be the goal there? I'd be wary of the try/catch swallowing underlying issues in the code. Also I believe code inside a try/catch can't be optimized as well so I'm worried about putting that in the core drawer code